src/connectors/index.ts

```diff
@@ -57,6 +57,31 @@ export const injected = new InjectedConnector({
   ],
 })
 
+export const coin98InjectedConnector = new InjectedConnector({
```

Comment:
what difference between `coin98InjectedConnector` and `injected` above. Why dont you use `injected`?

Reply:
```
injectConnectorParam = {
  supportedChainIds: [....]
}
export const injected = new InjectedConnector(injectConnectorParam)
export const coin98InjectedConnector = new InjectedConnector(injectConnectorParam)
```

Reply:
@TienPQ149